pegasus/test/test_curriculum_course.rb

```diff
@@ -32,7 +32,7 @@ def setup
 
     it 'should get all lessons' do
       assert_equal(18, @course.get_lessons.count)
-      assert(@course.get_lessons.first.is_a?(Lesson))
+      assert(@course.get_lessons.first.is_a?(CurriculumCourse::Lesson))
```

Comment:
I am just curious, did these change because of a test failure due to a conflict between the new Lesson class in Rails? I am surprised to see that pegasus tests might have access to rails classes. or did rubymine take care of this as part of an automated rename? either way, extra specificity seems fine.

Reply:
This was actually causing issues in dashboard because somehow the types defined in the `CurriculumCourse` file in the top-level namespace were overriding the ones defined in the model file sometimes. I don't remember the exact details, but this fixed it.